apps/minifront/src/components/staking/account/delegation-value-view/index.tsx

```diff
@@ -12,43 +13,46 @@ import { getValidatorInfoFromValueView } from '@penumbra-zone/getters';
  * https://github.com/penumbra-zone/penumbra/issues/3882, we may be able to
  * remove `votingPowerAsIntegerPercentage`.
  */
-export const DelegationValueView = ({
-  valueView,
-  votingPowerAsIntegerPercentage,
-  unstakedTokens,
-}: {
-  /**
-   * A `ValueView` representing the address's balance of the given delegation
-   * token.
-   */
-  valueView: ValueView;
-  votingPowerAsIntegerPercentage?: number;
-  /**
-   * A `ValueView` representing the address's balance of staking (UM) tokens.
-   * Used to show the user how many tokens they have available to delegate.
-   */
-  unstakedTokens?: ValueView;
-}) => {
-  const validatorInfo = getValidatorInfoFromValueView(valueView);
+export const DelegationValueView = memo(
```

Comment:
This `memo()` call is the only change to this file. Everything else is just indentation changes.